src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/CSharpFormatter.cs

```diff
@@ -48,26 +52,26 @@ internal class CSharpFormatter
             RazorCodeDocument codeDocument,
             Range range,
             Uri uri,
-            FormattingOptions options)
+            FormattingOptions options,
+            bool formatOnClient = false)
```

Comment:
You're keeping this around for now in the case you want to go to the back to client formatting right?

Reply:
Yeah